pkg/server/server.go

```diff
@@ -200,6 +202,33 @@ func Run(logger log.Logger) error {
 		func(error) { serverInsecure.Shutdown(context.Background()) },
 	)
 
+	{
```

Comment:
Because we add the logger with the component. We don't log anywhere after this, block, but if someone added any logging they would be surprised it has the component. It's just to scope it. Pattern is used in prometheus, and we stole it for stolon-pgbouncer.